interfaces/portal/src/app/components/page-layout-payment/components/start-payment/start-payment.component.html

```diff
@@ -0,0 +1,30 @@
+<p-button
+  label="Start payment"
+  i18n-label
+  (click)="openStartPaymentDialog()"
+  icon="pi pi-send"
+  [iconPos]="rtlHelper.createPosition('start')()"
+  rounded
+></p-button>
+
+<!-- ##TODO: don't use form-dialog for something that doesn't need input? -->
+<app-form-dialog
+  #startPaymentDialog
+  header="Start payment"
+  i18n-header
+  headerIcon="pi pi-send"
+  [mutation]="startPaymentMutation"
+  [mutationData]="undefined"
+>
+  <p>
+    <span i18n>Your payment has been approved by all needed approvers.</span>
```

Comment:
This line shouldn't be there yet in the current item, as we don't have approvers.